src/ci/run.sh

```diff
@@ -61,6 +61,12 @@ if [ "$DEPLOY$DEPLOY_ALT" != "" ]; then
   elif [ "$DEPLOY_ALT" != "" ]; then
     RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --enable-llvm-assertions"
   fi
+
+  if [ "$RUST_RELEASE_CHANNEL" = nightly ]; then
+      if [ "$MACOSX_DEPLOYMENT_TARGET" != "" ]; then
+          RUST_CONFIGURE_ARGS="$RUST_CONFIGURE_ARGS --enable-lldb"
+      fi
+  fi
```

Comment:
Instead of hardcoding this here, could [these lines](https://github.com/rust-lang/rust/blob/fb65d7563c93509c343a98e5cccf8e5ab4506924/.travis.yml#L80-L112) be modified to pass `--enable-lldb` via RUST_CONFIGURE_ARGS?

Reply:
My understanding was that for now we were only going to ship lldb in the nightly channel.  Is doing this in `.travis.yml` compatible with that?

Reply:
True yeah. We're pretty loose about "nightly only" components in the sense that we predominantly let them ride to stable an use "preview" in the component name to signal its instability